tests/renderRepoCard.test.js

```diff
@@ -255,4 +255,20 @@ describe("Test renderRepoCard", () => {
     expect(queryByTestId(document.body, "stargazers")).toBeNull();
     expect(queryByTestId(document.body, "forkcount")).toBeNull();
   });
+  
+  it("should render template", () => {
+    document.body.innerHTML = renderRepoCard({
+      ...data_repo.repository,
+      isTemplate: true
+    });
+    expect(queryByTestId(document.body, "template")).toBeDefined();
```

Comment:
I think we should change the data-testid to `template-badge`

Reply:
done